game/common/color.lua

```diff
@@ -0,0 +1,22 @@
+
+local COLOR = {}
```

Comment:
what about `domain/defs/color.lua`?

Reply:
It's kind of different, I guess? You think I should move it here?

Reply:
Hmm, I don't know? I guess it's ok like this.